common/scala/src/main/scala/org/apache/openwhisk/core/entity/ActivationResult.scala

```diff
@@ -45,6 +48,15 @@ protected[core] case class ActivationResponse private (val statusCode: Int, val
   def isSuccess = statusCode == ActivationResponse.Success
   def isApplicationError = statusCode == ActivationResponse.ApplicationError
   def isContainerError = statusCode == ActivationResponse.DeveloperError
+  def isContainerTimeoutError(timeout: FiniteDuration) = {
+    statusCode == ActivationResponse.DeveloperError &&
+    (result.get match {
```

Comment:
This will throw an error if the Option is `None`

Reply:
done, handled the case in the new method `isTimedoutActivation` in `WhiskActivation`